src/JaegerExporter.php

```diff
@@ -76,9 +76,10 @@ public function __construct($serviceName, array $options = [])
         ];
         $this->host = $options['host'];
         $this->port = (int) $options['port'];
+        $this->spanConverter = empty($options['spanConverter']) ? new SpanConverter() : $options['spanConverter'];
```

Comment:
1. Is there a need for the span converter to be injected? It's not being used for tests.

2. This needs an instance variable declaration above.